src/main/webapp/pipe.js

```diff
@@ -1,6 +1,15 @@
 function pipelineUtils() {
     var self = this;
     this.updatePipelines = function(divNames, errorDiv, view, fullscreen, page, component, showChanges, aggregatedChangesGroupingPattern, timeout, pipelineid, jsplumb) {
+
+        // Don't refresh pipelines if the tab/window is not active
+        if (document.hidden) {
```

Comment:
Is "document.hidden" supports in all browsers?

Reply:
@pskumar448 let's address that in a separate commit so that we can get this PR in